homeassistant/components/media_player/enigma2.py

```diff
@@ -0,0 +1,201 @@
+"""
```

Comment:
Rename this file to `enigma2/media_player.py`

Reply:
Thanks @balloob that is done now.